x-pack/plugins/lens/public/metric_visualization/expression.tsx

```diff
@@ -93,7 +93,9 @@ export const getMetricChartRenderer = (
   ) => {
     const resolvedFormatFactory = await formatFactory;
     ReactDOM.render(
-      <MetricChart {...config} formatFactory={resolvedFormatFactory} />,
+      <I18nProvider>
+        <MetricChart {...config} formatFactory={resolvedFormatFactory} />,
```

Comment:
I'm still getting an error in the console, but a different one:
<img width="662" alt="Screenshot 2020-11-02 at 16 43 31" src="https://user-images.githubusercontent.com/1508364/97887739-97f24100-1d2a-11eb-8180-50dc5e5e0a20.png">

It seems to be caused by the trailing comma in line 97 here - looks like it can be removed

Reply:
sorry!